microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md

```diff
@@ -59,15 +59,13 @@ Mobile devices have much less CPU power than desktops and laptops.  Whenever you
 
 1. In DevTools, open the **Performance** tool.
 
-1. Select the **Screenshots** checkbox.
-
 1. Click **Capture settings** (![Capture settings](./index-images/capture-settings-icon.png)).  DevTools reveals settings related to how it captures performance metrics.
 
 1. For **CPU**, select **4x slowdown**.  DevTools throttles your CPU so that it's 4 times slower than usual.
 
    ![CPU throttle](./index-images/capture-settings.png)
```

Comment:
the screenshot shows the Screenshots checkbox selected, not cleared.  Clear it in the screenshot?  Is this checkbox  selected by default?

Reply:
checkbox doesn't matter